INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:40:10] "GET /api/v1.0/v1.0/index HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:40:20] "GET /api/v1.0/v1.0/index HTTP/1.1" 404 -
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO demo.py:10 info
DEBUG demo.py:11 debug
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:40:26] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO demo.py:10 info
DEBUG demo.py:11 debug
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:41:57] "GET /api/v1.0/index HTTP/1.1" 200 -
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO demo.py:10 info
DEBUG demo.py:11 debug
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:41:58] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 11:48:43] "POST /findByAddressEntBasic HTTP/1.1" 404 -
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO demo.py:10 info
DEBUG demo.py:11 debug
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 12:32:51] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO demo.py:10 info
DEBUG demo.py:11 debug
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 15:03:42] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR demo.py:8 xxx严重错误
WARNING demo.py:9 警告
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 15:04:12] "GET /api/v1.0/index HTTP/1.1" 200 -
ERROR __init__.py:97 Error: Directory migrations already exists
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR __init__.py:97 Error: Directory migrations already exists
ERROR __init__.py:97 Error: Directory migrations already exists
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:07:05] "GET /18385855845?image_code=1 HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:07:05] "GET /favicon.ico HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:07:22] "GET /sms_codes/18385855845?image_code=1 HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:07:42] "GET /api/v1.0/sms_codes/18385855845?image_code=1 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 14:09:02] "GET /api/v1.0/image_codes/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\constants.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 15:53:32] "POST /findRealtimeEntBasic/notes HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 15:53:37] "POST /findRealtimeEntBasic/notes HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [17/May/2019 15:53:51] "POST /findRealtimeEntBasic/notes HTTP/1.1" 404 -
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:29] "GET /index HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:29] "GET /favicon.ico HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:33] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:37] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:38] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:38] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:38] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:47:38] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:48:24] "GET /users HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:48:27] "GET /users HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:52:52] "GET /register HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 09:53:12] "GET /register.html HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\constants.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\constants.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:07:46] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:07:56] "GET /api/v1.0/index.html HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:36:24] "GET /api/v1.0/users HTTP/1.1" 405 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:36:27] "GET /api/v1.0/users HTTP/1.1" 405 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:36:27] "GET /api/v1.0/users HTTP/1.1" 405 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:44:48] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:44:48] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:51:07] "GET /api/v1.0/users HTTP/1.1" 405 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:51:07] "POST /api/v1.0/image_codes/1 HTTP/1.1" 405 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:51:07] "GET /api/v1.0/image_codes/1 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:56:16] "GET /api/v1.0/image_codes/1 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:56:16] "GET /api/v1.0/image_codes/1 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 11:56:16] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:33] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:34] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:35] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:35] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:36] "GET /api/v1.0/register.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/css/ihome/main.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/css/ihome/register.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/js/ihome/register.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:46] "GET /api/v1.0/image_codes/bc03b067-c9f4-48ed-b421-81716cffd674 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:58] "GET /api/v1.0/image_codes/c663ca80-d8c9-4437-8932-905c37684bbc HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:31:59] "GET /api/v1.0/image_codes/b5ddd105-a0f8-4f7d-a399-40c192eb6d46 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 12:32:00] "GET /api/v1.0/image_codes/69d8437b-80c2-4a60-8d03-32a8960becfe HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\web_html.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\web_html.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:09:20] "GET /api/v1.0/image_codes/e4d0112c-16e9-4eae-b98d-cb0b04ad9dbf HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:37] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:37] "GET /api/v1.0/image_codes/cce6cf5d-1788-48cf-a253-b1e0553f6a60 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:41] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:41] "GET /api/v1.0/image_codes/698d1885-da63-43cb-a939-0d2d88f9cc0b HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:44] "GET /api/v1.0/image_codes/7ccd572a-1372-4916-9a44-f20eee343938 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:15:45] "GET /api/v1.0/image_codes/57edd2e8-dbfe-41ca-9d7b-5728c79f0896 HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:20] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:21] "GET /api/v1.0/image_codes/4f405989-836f-4d28-aab0-4bda73708109 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:27] "GET /api/v1.0/image_codes/1ee451ab-aa79-4122-aa00-7a81d1cfb95f HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:28] "GET /api/v1.0/image_codes/895134d6-d4f0-427a-8994-e805fa37e71a HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:29] "GET /api/v1.0/image_codes/2a7f3b31-0973-4a1c-a626-8c01692df3e3 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:29] "GET /api/v1.0/image_codes/62b0ae37-f309-4300-8fef-ff930fb66c8b HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:30] "GET /api/v1.0/image_codes/d7cd84bf-b0d1-494a-abfe-02d7ec831e86 HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:30] "GET /api/v1.0/image_codes/de9199e8-9b6b-4a73-b8fe-dd16143603cc HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:30] "GET /api/v1.0/image_codes/b6fc93d8-cada-4618-b08a-95668135397a HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:30] "GET /api/v1.0/image_codes/bdfe00da-8f24-4fa7-9b89-a7bbcd639b8a HTTP/1.1" 500 -
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:28:40] "GET /api/v1.0/image_codes/a6b267f3-6686-4ba2-8a27-f3b01dbd8bea HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:32:33] "GET /api/v1.0/image_codes/18e500f5-b6b0-4859-a7df-532fe645c8d6 HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:33:01] "GET /api/v1.0/image_codes/32c99fad-be0d-453f-b2b4-d810f7c1daf3 HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:36:23] "GET /api/v1.0/image_codes/3f82001d-0ab6-4637-a32a-40c1adfba403 HTTP/1.1" 500 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:56] "GET /register.html HTTP/1.1" 304 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:56] "GET /api/v1.0/image_codes/5acf6408-35ff-4935-91b0-9a8793d3c4c9 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:56] "GET /api/v1.0/image_codes/041b3700-c533-4195-a7ae-3dfe56d9886e HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:57] "GET /api/v1.0/image_codes/a3fdfdf1-35c1-4fa0-b668-131b70714957 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:58] "GET /api/v1.0/image_codes/7a9a23d6-4704-4842-b588-08e4deeaa88d HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:58] "GET /api/v1.0/image_codes/ff48eb3e-8941-4462-82f8-4423aa53ac28 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:37:58] "GET /api/v1.0/image_codes/e3107ca9-1146-423a-85da-ba1e7797b74c HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:42:03] "GET /api/v1.0/image_codes/716cbb94-e8d0-46a7-93c6-4fde2e78a636 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:42:04] "GET /api/v1.0/image_codes/e8e53beb-ae69-4018-8df0-589e8e257cc1 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\xmltojson.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:48:42] "GET /api/v1.0/sms_codes/18384291574?image_code=et4j&image_code_id=e8e53beb-ae69-4018-8df0-589e8e257cc1 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:48:45] "GET /api/v1.0/image_codes/11e1dc37-a6cb-4a53-9482-89cba48f2b8b HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:48:56] "GET /api/v1.0/sms_codes/18384291574?image_code=jkc8&image_code_id=11e1dc37-a6cb-4a53-9482-89cba48f2b8b HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:48:58] "GET /api/v1.0/image_codes/c5c249a9-64ae-4731-94ec-eef076f3227e HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:49:04] "GET /api/v1.0/sms_codes/18384291574?image_code=6cxe&image_code_id=c5c249a9-64ae-4731-94ec-eef076f3227e HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:51:55] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:51:59] "GET /api/v1.0/sms_codes/18384291574?image_code=6cxe&image_code_id=c5c249a9-64ae-4731-94ec-eef076f3227e HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:52:12] "GET /api/v1.0/image_codes/6eea5f8d-169b-4170-8eda-6602c83e5f49 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:52:24] "GET /api/v1.0/sms_codes/18384291574?image_code=pok5&image_code_id=6eea5f8d-169b-4170-8eda-6602c83e5f49 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:52:44] "GET /api/v1.0/sms_codes/18384291574?image_code=pok5&image_code_id=6eea5f8d-169b-4170-8eda-6602c83e5f49 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:53:23] "GET /api/v1.0/image_codes/124fa965-4e04-4a99-9f50-32d040c47c57 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:55:40] "GET /api/v1.0/sms_codes/18384291574?image_code=hear&image_code_id=124fa965-4e04-4a99-9f50-32d040c47c57 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:56:38] "GET /api/v1.0/sms_codes/18384291574?image_code=hear&image_code_id=124fa965-4e04-4a99-9f50-32d040c47c57 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:57:56] "GET /api/v1.0/image_codes/a8bf1865-eda1-4001-8060-4f2305f0343c HTTP/1.1" 200 -
ERROR verify_code.py:81 incomplete format
ERROR verify_code.py:117 send_template_sms() missing 1 required positional argument: 'temp_id'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 13:58:03] "GET /api/v1.0/sms_codes/18384291574?image_code=rf7m&image_code_id=a8bf1865-eda1-4001-8060-4f2305f0343c HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:02:35] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:02:55] "GET /api/v1.0/image_codes/58c1a111-0dba-422f-ae4c-c1861778f092 HTTP/1.1" 200 -
ERROR verify_code.py:81 incomplete format
ERROR verify_code.py:117 send_template_sms() missing 1 required positional argument: 'temp_id'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:03:15] "GET /api/v1.0/sms_codes/18384291574?image_code=98IR&image_code_id=58c1a111-0dba-422f-ae4c-c1861778f092 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:03:21] "GET /api/v1.0/sms_codes/18384291574?image_code=98IR&image_code_id=58c1a111-0dba-422f-ae4c-c1861778f092 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:03:57] "POST /api/v1.0/users HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:08:26] "GET /api/v1.0/image_codes/b50b9b99-4693-46a2-9cbc-7c83e31aa259 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:08:31] "GET /api/v1.0/sms_codes/18384291574?image_code=fsbt&image_code_id=b50b9b99-4693-46a2-9cbc-7c83e31aa259 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:09:29] "GET /api/v1.0/sms_codes/18384291574?image_code=fsbt&image_code_id=b50b9b99-4693-46a2-9cbc-7c83e31aa259 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:10:42] "GET /api/v1.0/image_codes/67627001-0768-4ee0-bd56-66892a97bf65 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:10:48] "GET /api/v1.0/sms_codes/18384291574?image_code=awqb&image_code_id=67627001-0768-4ee0-bd56-66892a97bf65 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:13:10] "GET /api/v1.0/image_codes/ca482793-a9c5-4ef0-9ae4-b7cb4d81c016 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:13:17] "GET /api/v1.0/sms_codes/18384291574?image_code=pl3o&image_code_id=ca482793-a9c5-4ef0-9ae4-b7cb4d81c016 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:14:28] "GET /api/v1.0/sms_codes/18384291574?image_code=pl3o&image_code_id=ca482793-a9c5-4ef0-9ae4-b7cb4d81c016 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:16:33] "GET /api/v1.0/sms_codes/18384291574?image_code=pl3o&image_code_id=ca482793-a9c5-4ef0-9ae4-b7cb4d81c016 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:16:42] "GET /api/v1.0/image_codes/d6275a32-f049-4a06-97b7-1f1813ad3cd5 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:16:49] "GET /api/v1.0/sms_codes/18384291574?image_code=pw5y&image_code_id=d6275a32-f049-4a06-97b7-1f1813ad3cd5 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:22:48] "GET /api/v1.0/image_codes/2ced04ae-a5ab-476e-9f16-da5a40674e25 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:22:54] "GET /api/v1.0/sms_codes/18384291574?image_code=flax&image_code_id=2ced04ae-a5ab-476e-9f16-da5a40674e25 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:24:52] "GET /api/v1.0/image_codes/d9441c6e-3ed8-4a40-ad43-2c8998584bf4 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:24:58] "GET /api/v1.0/sms_codes/18384291574?image_code=koxw&image_code_id=d9441c6e-3ed8-4a40-ad43-2c8998584bf4 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:27:08] "GET /api/v1.0/image_codes/abfec08a-2c27-4484-b0cd-30a03fe37a99 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:27:14] "GET /api/v1.0/sms_codes/18384291574?image_code=zhix&image_code_id=abfec08a-2c27-4484-b0cd-30a03fe37a99 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:27:39] "GET /api/v1.0/image_codes/b8c86515-5ad4-444f-b0ab-24320181f5b0 HTTP/1.1" 200 -
ERROR verify_code.py:83 incomplete format
ERROR verify_code.py:119 '<=' not supported between instances of 'str' and 'int'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:27:45] "GET /api/v1.0/sms_codes/18384291574?image_code=mftu&image_code_id=b8c86515-5ad4-444f-b0ab-24320181f5b0 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\CCPRestSDK.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:56:36] "GET /api/v1.0/image_codes/c6a58d3c-bb99-43f2-abd5-ca782317fcf9 HTTP/1.1" 200 -
ERROR verify_code.py:81 incomplete format
ERROR verify_code.py:117 '<=' not supported between instances of 'str' and 'int'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 14:56:43] "GET /api/v1.0/sms_codes/18384291574?image_code=eqhh&image_code_id=c6a58d3c-bb99-43f2-abd5-ca782317fcf9 HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:18:06] "GET /api/v1.0/image_codes/ed6cf2f7-dc3d-4a76-8bdf-18f4d15e9a98 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:18:09] "GET /api/v1.0/image_codes/bcb81a33-a8ff-41d0-96b3-ca9b3edf5e7c HTTP/1.1" 200 -
ERROR verify_code.py:81 incomplete format
ERROR verify_code.py:117 '<=' not supported between instances of 'str' and 'int'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:18:16] "GET /api/v1.0/sms_codes/18384291574?image_code=u4cj&image_code_id=bcb81a33-a8ff-41d0-96b3-ca9b3edf5e7c HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\libs\\yuntongxun\\sms.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:19:25] "GET /api/v1.0/image_codes/3bd0f097-b141-4360-b36e-779def160e3d HTTP/1.1" 200 -
ERROR verify_code.py:81 incomplete format
ERROR verify_code.py:117 '<=' not supported between instances of 'str' and 'int'
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:19:34] "GET /api/v1.0/sms_codes/18384291574?image_code=cxau&image_code_id=3bd0f097-b141-4360-b36e-779def160e3d HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\verify_code.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\postport.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:30:31] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:30:37] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:30:43] "GET /api/v1.0/image_codes/e855d4d0-1179-4344-b488-797bf2e584d8 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:30:49] "GET /api/v1.0/sms_codes/18384291574?image_code=edcz&image_code_id=e855d4d0-1179-4344-b488-797bf2e584d8 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:30:56] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\postport.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:10] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:13] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:13] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:13] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:14] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:14] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:14] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:14] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:14] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:15] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:15] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:15] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:15] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:16] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:16] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:16] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:16] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:16] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:17] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:17] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:17] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:18] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:18] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:19] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:32:24] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\postport.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:23] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:25] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:25] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:26] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:26] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:29] "GET /api/v1.0/image_codes/4f942aed-6f94-483a-b821-48578be26559 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:34:32] "POST /api/v1.0/users HTTP/1.1" 400 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\postport.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:38:46] "GET /api/v1.0/users HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\api_1_0\\postport.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [20/May/2019 15:40:14] "GET /api/v1.0/users HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\models.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\models.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 158-954-805
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'E:\\datum\\ihome_demo\\ihome\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
